torch/utils/pytree/__init__.py

```diff
@@ -33,12 +33,6 @@
 
 
 __all__ = [
-    "PyTree",
-    "FlattenFunc",
-    "UnflattenFunc",
-    "FlattenWithKeysFunc",
-    "ToDumpableContextFunc",
-    "FromDumpableContextFunc",
```

Comment:
These type aliases are removed from `__all__`. Otherwise, the public API test will ask to add these to `test/allowlist_for_publicAPI.json`.